common/lib/xmodule/xmodule/combined_open_ended_module.py

```diff
@@ -515,3 +528,23 @@ def non_editable_metadata_fields(self):
     # Proxy to CombinedOpenEndedModule so that external callers don't have to know if they're working
     # with a module or a descriptor
     child_module = module_attr('child_module')
+
+    def validate(self):
+        """
+        Validates the state of this instance. This is the override of the general XBlock method,
+        and it will also ask its superclass to validate.
+        """
+        validation = super(CombinedOpenEndedDescriptor, self).validate()
+        validation = StudioValidation.copy(validation)
+
+        _ = self.runtime.service(self, "i18n").ugettext  # pylint: disable=redefined-outer-name
+
+        validation.summary = StudioValidationMessage(
+            StudioValidationMessage.ERROR,
+            _(u"ORA 1 has been deprecated and removed from {0}. "
+              u"Please delete this component and replace with an "
+              u"ORA 2 component.".format(microsite.get_value('platform_name', settings.PLATFORM_NAME))
```

Comment:
Small changes:

ORA1 has been deprecated and removed from {0}. To use this assessment, replace this ORA1 component with an ORA2 component.